ihmc-perception/src/main/java/us/ihmc/perception/streaming/ROS2SRTVideoStreamImageMessageRelayWorker.java

```diff
@@ -1,26 +1,38 @@
 package us.ihmc.perception.streaming;
 
+import org.bytedeco.javacpp.BytePointer;
 import org.bytedeco.opencv.opencv_core.Mat;
 import perception_msgs.msg.dds.ImageMessage;
 import us.ihmc.communication.packets.MessageTools;
 import us.ihmc.communication.ros2.ROS2Helper;
 import us.ihmc.communication.ros2.ROS2SRTStreamTopicPair;
 import us.ihmc.perception.CameraModel;
 import us.ihmc.perception.RawImage;
+import us.ihmc.perception.cuda.CUDAJPEGProcessor;
 import us.ihmc.perception.imageMessage.CompressionType;
 import us.ihmc.perception.imageMessage.PixelFormat;
 import us.ihmc.perception.opencv.OpenCVTools;
 import us.ihmc.perception.tools.PerceptionMessageTools;
 import us.ihmc.ros2.ROS2Node;
 import us.ihmc.ros2.ROS2PublisherBasics;
 
+import javax.annotation.Nullable;
 import java.time.Instant;
 
 public class ROS2SRTVideoStreamImageMessageRelayWorker
 {
+   /**
+    * Set to true to enable compression to JPEG (using nvJPEG) before publishing the ImageMessage.
+    * Introduces GPU overhead and more latency, but useful for recording rosbags.
+    */
+   private static final boolean COMPRESS_USING_NVJPEG = true;
```

Comment:
It would be better if this was a JVM parameter or literal parameter in the constructor. Making options that require local code changes sucks, especially on the OCU, there's a bunch of garbage left afterwards.

Reply:
Yeah, having random changes on the OCU sucks. I always struggle setting the JVM parameters correctly though. I'd be fine with keeping it a static field since we would hardly ever change it. @ds58 What do you think about this?

Reply:
I do not see the benefit of using environment variables from the way we run the applications always from IDE